portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,52 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Shaunna!");
+    String pageOne = "Homepage with pictures. U/C";
+    String pageTwo = "List of projects in a resume way. U/C";
+    String pageThree = "Hidden Talents. U/C";
+
+
+    ArrayList<String> arr = new ArrayList<String>();
+    arr.add(pageOne);
```

Comment:
I think it would be fine to put the string inside the add, i.e.

`arr.add("Homepage with pictures. U/C")`